DAL/user.DAL.js

```diff
@@ -0,0 +1,22 @@
+const User = require("./../modules/user.module");
+
+const addNewUser = async (user) => {
+  const newUser = User.create(user);
+  return newUser;
+};
+
+const getUserByEmail = async (req, res) => {
+  const foundUser = User.findOne({ email: req.body.email });
```

Comment:
`req.body.email` what if the body doesn't have an email in it? protect this case.

Reply:
DONE